packages/react-router-dom/__tests__/data-browser-router-test.tsx

```diff
@@ -37,6 +37,9 @@ import {
   useLocation,
   createRoutesFromElements,
 } from "react-router-dom";
+import { isSubmitterElement } from "../dom";
+
+fixSubmitEventSubmitter(window);
```

Comment:
If we were to put this in another file, would we dup the listeners on `window`?  Wondering if it should either return an `unlisten` and be called in a `beforeEach`/`afterEach` setup?  Or - could this just move to `setup.ts` and be applied globally for all `react-router-dom` tests?

Reply:
From what I can tell of how jsdom/rtl are wired up, it looks like we just need to do it once per test suite/file, rather than per test. And since jest runs each test file in an isolated worker, I don't believe any cleanup should be needed. I like the setup.ts idea, I'll move it there 👍

Reply:
Done